packages/spindle-hooks/package.json

```
@@ -21,6 +21,7 @@
     "build:esm": "run-s build:esm:js build:esm:rename",
     "build:esm:js": "tsc -p tsconfig.esm.json",
     "build:esm:rename": "npx renamer --find js --replace mjs 'dist/**'",
+    "build:esm:extensionReplace": "jscodeshift -t ../../bin/outputExtensionReplace.js dist/** --extensions=mjs",
```

```suggestion
    "build:esm": "run-s build:script:esm:*",
    "build:esm:js": "tsc -p tsconfig.esm.json",
    "build:esm:rename": "npx renamer --find js --replace mjs 'dist/**'",
    "build:esm:extensionReplace": "jscodeshift -t ../../bin/outputExtensionReplace.js dist/** --extensions=mjs",
```

build:esm内で呼ぶか、*で指定した方が良さそう？

---

あ、Suggestion間違ってるのでお待ちを

---

修正済み ✅

---

おっと、ここだけ個別指定だったんですね。採用させていただきます:pray: